lib/Jsrt/Core/JsrtCore.cpp

```diff
@@ -887,13 +891,17 @@ CHAKRA_API JsCreateString(
     }
 
     return ContextAPINoScriptWrapper([&](Js::ScriptContext *scriptContext, TTDRecorder& _actionEntryPopper) -> JsErrorCode {
-
         Js::JavascriptString *stringValue = Js::LiteralStringWithPropertyStringPtr::
-            NewFromCString(content, (CharCount)length, scriptContext->GetLibrary());
-
+        NewFromCString(content, (CharCount)length, scriptContext->GetLibrary());
```

Comment:
These calls should not happen on the length = 0 path, may be the cause of the test failures